ndpi-netfilter/src/Makefile

```diff
@@ -2,9 +2,10 @@
 NDPI_SRC := ../../src
 NDPI_PRO := ${NDPI_SRC}/lib/protocols
 
-ccflags-y += -I${src}/${NDPI_SRC}/include -I${src}/${NDPI_SRC}/lib -I${src}/../libre -I${src}/${NDPI_SRC}/lib/third_party/include 
-ccflags-y += -DHAVE_CONFIG_H -DNDPI_LIB_COMPILATION -DOPENDPI_NETFILTER_MODULE -DNDPI_DETECTION_SUPPORT_IPV6 -g 
+ccflags-y += -I${src}/${NDPI_SRC}/include -I${src}/${NDPI_SRC}/lib -I${src}/../libre -I${src}/${NDPI_SRC}/lib/third_party/include
+ccflags-y += -DHAVE_CONFIG_H -DNDPI_LIB_COMPILATION -DOPENDPI_NETFILTER_MODULE -DNDPI_DETECTION_SUPPORT_IPV6 -g
 ccflags-y += -Wno-declaration-after-statement
+ccflags-y += -Wno-strict-prototypes
```

Comment:
`-Wno-strict-prototypes` is added here to disable a warning (as error) when compiling `ndpi_main.c` with clang.
https://github.com/vel21ripn/nDPI/blob/b19e6392cf0c7d51c44e076a91fc4db0cbbd6403/src/lib/ndpi_main.c#L232-L234
https://github.com/vel21ripn/nDPI/blob/b19e6392cf0c7d51c44e076a91fc4db0cbbd6403/src/lib/ndpi_main.c#L8135-L8137
https://github.com/vel21ripn/nDPI/blob/b19e6392cf0c7d51c44e076a91fc4db0cbbd6403/src/lib/ndpi_main.c#L8183-L8185
https://github.com/vel21ripn/nDPI/blob/b19e6392cf0c7d51c44e076a91fc4db0cbbd6403/src/lib/ndpi_main.c#L8203-L8205